### PR TITLE
Adrian's comments

### PR DESCRIPTION
**Pull request by QiufangMa**
This PR resolves Adrian's comments (https://github.com/boucadair/policy-based-network-acl/issues/15).
One comment left to be done:
"It would be good if the document title indicated (as the Abstract does)
  what the document contains.  Something like...
  Management Tools for Policy-based Access Control"

### CODE REVIEW

**Review by boucadair — APPROVED** [review]

**Comment by QiufangMa** [comment]
I am thinking the following about the title:
1. An integrated management solution for policy based network access control
2. A YANG data model, reference architecture and Raidus extension for policy based network access control
3. Management tools for policy based network access control(suggested by Adrian, but i am unsure i like the word "tools")

Thoughts and suggestions?

**Comment by danielkinguk** [comment]
How about... 

"A YANG data model and RADIUS extension for Policy-Based Network Access Control"

Although, I get grumpy with even using "policy-based" . Oh, in fact, how much "policy" is discussed/defined in the document? We don't define, or provide a reference in the document.

**Comment by boucadair** [comment]
> "A YANG data model and RADIUS extension for Policy-Based Network Access Control"

I like this one as well.

**Comment by QiufangMa** [comment]
> > "A YANG data model and RADIUS extension for Policy-Based Network Access Control"
> 
> I like this one as well.

okay, let's use this one, I will check with Adrian. Regarding the "policy-based" term, let's use RFC2753 "Framework for Policy-bsed Admission Control" for reference. Good for you?

**Comment by danielkinguk** [comment]
Just poked Adrian (he is sat next to me), title is good. :-)

File: draft-ietf-opsawg-ucl-acl.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "A Policy-based Network Access Control"
+title: "A YANG data model and RADIUS extension for Policy-based Network Access Control"
```
Comment (boucadair) [inline draft-ietf-opsawg-ucl-acl.md:2]:
```suggestion
title: "A YANG Data Model and RADIUS Extension for Policy-based Network Access Control"
```

**Pull request merged**